ui/src/types/auth.ts

```diff
@@ -10,4 +10,5 @@ export type Auth0Config = {
   clientID: string
   domain: string
   redirectURL: string
+  socialSignUpOn: boolean
```

Comment:
nit: `toggled` might be more idiotmatic / conventional than `on` eg `socialSignUpToggled`.

Reply:
Such a great suggestion. Owing to the fact that this is set on IDPE and k8s already as is (and will be a temporary property), I think it would be easiest to keep this naming as is for the time being. But I'll definitely bear in mind that boolean values should have more well-defined names